arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,1020 @@
+	};
+
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	memory@80000000 {
+                device_type = "memory";
+		reg = <0x80000000 0x20000000>;
+	};
+
+	led-controller {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+
+		led-0 {
+			color = <LED_COLOR_ID_YELLOW>;
+			label = "yellow-led";
+			gpios = <&gpio4 23 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-1 {
+			color = <LED_COLOR_ID_RED>;
+			label = "red-led";
+			gpios = <&gpio4 24 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-2 {
+			color = <LED_COLOR_ID_GREEN>;
+			label = "green-led";
+			gpios = <&gpio4 25 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led-3 {
+			color = <LED_COLOR_ID_BLUE>;
+			label = "blue-led";
+			gpios = <&gpio4 26 GPIO_ACTIVE_HIGH>;
+			default-state = "off";
+		};
+
+		/* The following are meant to be userspace controlled IO. The
+		 * kernel does not allow setting default state of GPIO, but
+		 * the default state of LEDs can be set, which is why the LED
+		 * subsystem is used.
+		 * Additionally, Linux regulators often cannot be manually
+		 * controlled, making LED the ideal way to control certain pins
+		 */
+		 en-emmc {
+			label = "en-emmc";
+			gpios = <&gpio4 22 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		 en-sd {
+			label = "en-sd";
+			gpio = <&gpio6 14 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+	};
+
+		reg_gpio_dvfs: regulator-gpio {
+			compatible = "regulator-gpio";
+			pinctrl-names = "default";
+			pinctrl-0 = <&pinctrl_dvfs>;
+			regulator-min-microvolt = <1300000>;
+			regulator-max-microvolt = <1400000>;
+			regulator-name = "gpio_dvfs";
+			regulator-type = "voltage";
+			gpios = <&gpio5 3 GPIO_ACTIVE_HIGH>;
+			states = <1300000 0x1 1400000 0x0>;
+		};
+
+		reg_adc_vref: adc_vref {
+			compatible = "regulator-fixed";
+			regulator-name = "ADC_VREF";
+			regulator-min-microvolt = <2500000>;
+			regulator-max-microvolt = <2500000>;
+			//gpio = EN_2.5_VREF driven by Silabs;
+			//enable-active-high;
+		};
+
+		reg_flexcan_3v3: en-can@0 {
+			compatible = "regulator-fixed";
+			regulator-name = "en-can";
+			regulator-min-microvolt = <3000000>;
+			regulator-max-microvolt = <3000000>;
+			gpio = <&gpio6 20 GPIO_ACTIVE_HIGH>;
+		};
+
+	reg_3v3: regulator-3v3 {
+		compatible = "regulator-fixed";
+		regulator-name = "3V3";
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+		regulator-always-on;
+		pinctrl-0 = <>; /* still needed if always-on? */
+	};
+
+        i2c_gpio@0 {
+	        compatible = "i2c-gpio";
+	        clock-frequency = <100000>;
+	        pinctrl-names = "default";
+	        pinctrl-0 = <&pinctrl_i2c_gpio>;
+		gpios = <
+			&gpio3 9 GPIO_ACTIVE_HIGH /* I2C_DAT on HD12 pin 1 */
+			&gpio3 8 GPIO_ACTIVE_HIGH /* I2C_CLK on HD12 pin 6 */
+			>;
+	        status = "okay";
+		i2c-gpio,delay-us = <2>;	/* ~100 kHz */
+	        #address-cells = <1>;
+	        #size-cells = <0>;
+        };
+
+	pps {
+		compatible = "pps-gpio";
```

Comment:
Also, need to make sure this driver is present in both defconfigs, I am not sure if it is or not but I don't think so as I think this is the first 6ul we've rolled this out on.